sites/public/pages/listings.tsx

```diff
@@ -175,6 +181,28 @@ const ListingsPage = () => {
               errorMessage={t("errors.multipleZipCodeError")}
               defaultValue={filterState?.zipcode}
             />
+            <label className="field-label">Rent Range</label>
+            <div className="flex flex-row">
+              <Field
+                id="minRent"
+                name={ListingFilterKeys.minRent}
+                register={register}
+                type="number"
+                placeholder={t("t.min")}
+                prepend="$"
+                defaultValue={filterState?.minRent}
+              />
+              <div className="flex items-center">{t("t.to")}</div>
```

Comment:
Smallest nit ever: when a user tabs onto the fields, the highlight kind of crowds the "to". Could we slightly increase the padding around it?

![image](https://user-images.githubusercontent.com/1295824/132057714-d2b7d8b0-024d-464d-89a9-0aa01a32ee72.png)

Reply:
Added padding: 
![image](https://user-images.githubusercontent.com/83078310/132061549-d6bab7e8-c461-427c-9e10-93a126bab63f.png)